config/spring19_seminar/cola_var/cola_bert_ccg.conf

```diff
@@ -0,0 +1,77 @@
+// Import the defaults using the relative path
+include "../defaults.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
+exp_name = exp_pretrain
+run_name = run_bert_ccg
+
+
+// Logistics
+cuda = 0
+random_seed = 42
```

Comment:
Thanks! Only override things you really need to, so it's easier to sep differences between setups—I think these can just stay in defaults.conf, for example.

Reply:
@sleepinyourhat  in the updated config files the default settings have been removed